src/api/spec/factories/users.rb

```diff
@@ -49,6 +54,13 @@
           create(:service_token, user: user)
         end
       end
+
+      factory :non_recently_logged_user do
```

Comment:
Do we really need a factory for this if we only use it once?

Reply:
I think we can use this factory elsewhere too. I would keep it

Reply:
![F**K THE FUTURE SIGN](https://user-images.githubusercontent.com/514785/60801081-a5b7a680-a176-11e9-96e7-b3403037c02c.jpeg)

Reply:
is that a blocking for you @hennevogel ?